File: back/src/main/java/org/bliz/trivial/controller/model/Answer.java
package org.bliz.trivial.controller.model;

import java.io.Serializable;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Answer implements Serializable {

	private static final long serialVersionUID = -7987994210781690946L;

	private Integer answerCode;

	private Integer isCorrect;

	private String text;

	private Question question;
}

File: front/src/app/models/answer.ts
import { Question } from './question';
export class Answer {

    answerCode: number;
    isCorrect: number;
    text: string;
    question: Question;

    constructor(answerCode: number, isCorrect: number, text: string, question: Question) {
        this.answerCode = answerCode;
        this.isCorrect = isCorrect;
        this.text = text;
        this.question = question;
    }
}